gradle/dependencies.gradle

```diff
@@ -102,20 +104,25 @@ ext {
             commonsIO               : "commons-io:commons-io:${version.commonsIO}",
             robolectric             : "org.robolectric:robolectric:${version.robolectric}",
             json                    : "org.json:json:${version.jsonVersion}",
+            androidxTestKtx         : "androidx.test:core-ktx:${version.androidxTestKtx}",
+            mockk                   : "io.mockk:mockk:${version.mockk}",
 
             // play services
             gmsLocation             : "com.google.android.gms:play-services-location:${version.gmsLocation}",
-            errorprone              : "com.google.errorprone:error_prone_core:${version.errorprone}"
+            errorprone              : "com.google.errorprone:error_prone_core:${version.errorprone}",
+
+            kotlinxSerializationJson: "org.jetbrains.kotlinx:kotlinx-serialization-json:1.7.3"
```

Comment:
We need a version variable here as well :)

Reply:
Is changed ✅